test/recipes/25-test_x509.t

```diff
@@ -124,8 +124,12 @@ sub test_errors { # actually tests diagnostics of OSSL_STORE
     return $res && $found;
 }
 
+# 3 tests for non-existence of spurious OSSL_STORE ASN.1 parse error output.
+# This requires provoking somehow a non-successful late exit of the app.
```

Comment:
I'd drop the _somehow_ here.

Reply:
I've just pushed a fixup that drops "somehow" and adds a little more detail:
```
# This requires provoking a failure exit of the app after reading input files.
```